sample/java/src/main/java/com/datadog/android/sample/MainActivity.java

```diff
@@ -79,6 +84,7 @@ protected void onRestart() {
 
     @Override
     protected void onResume() {
+        mRumMonitor.startView(this, getComponentName().getClassName(), new HashMap<String, Object>());
```

Comment:
you do not want the .getCannonicalName() ?

Reply:
It's a test for now, eventually it will be done automatically with activity instrumentation ;)